lib/xchain/provider/fetch.go

```diff
@@ -444,6 +444,8 @@ func getConsXBlock(ctx context.Context, ref xchain.EmitRef, cprov cchain.Provide
 		return xchain.Block{}, errors.Wrap(err, "fetch consensus xblock")
 	} else if !ok {
 		return xchain.Block{}, errors.New("no consensus xblocks [BUG]")
+	} else if len(xblock.Msgs) == 0 {
+		return xchain.Block{}, errors.New("no messages [BUG]")
```

Comment:
nit: `empty consensus xblock [BUG]`

"no messages" is just a bit vauge, what has no messages?